### DISCUSSION TITLE
Specifying slide elements not as child of slide-wrapper

### DISCUSSION THREAD

**Discussion by titantwentyone** [nolimits4web/swiper#4519] [Q&A]
I have some markup for Swiper such that:

```
<div class="swiper-container">
    <div class="swiper-wrapper">
        <div style="display:contents">
            <div class="swiper-slide"></div>
            <div class="swiper-slide"></div>
            <div class="swiper-slide"></div>
        </div>
    </div>
</div>
```

Is there a way to let Swiper know to ignore the "display:contents" div and treat the "swiper-slide" elements as slides. Vanilla usage seems to imply that "swiper-slide" must be a direct child of "swiper-wrapper" and I haven't been able to find a way to change this,

**Comment by nolimits4web** [comment]
No sorry, such behavior is not supported

**Reply by titantwentyone** [reply]
Thanks for the response. My use case is that the CMS I am using outputs a "dummy element" which is rendered when repeating elements from a DB. As such, this dummy element seperates the wrapper from the slide elements. Would this be something for consideration in the future? I was an avid user of Cycles2 a while back and I believe that had this ability.
